llvm/tools/llvm-profgen/llvm-profgen.cpp

```diff
@@ -189,5 +189,7 @@ int main(int argc, const char *argv[]) {
     Generator->write();
   }
 
+  PerfScriptReader::removeTempFiles();
+
   return EXIT_SUCCESS;
```

Comment:
Nit: there are more "return EXIT_SUCCESS;"  in other places, may be useful to also cover them(like use/call a wrapper function)? though other usages are not commonly used..

Reply:
Using CleanupInstaller can avoid this since cleanup will be auto run in destructor of static variable.